validator/build.py

```diff
@@ -164,11 +164,11 @@ def GenValidatorProtoascii(out_dir):
   assert re.match(r'^[a-zA-Z_\-0-9]+$', out_dir), 'bad out_dir: %s' % out_dir
 
   protoascii_segments = [open('validator-main.protoascii').read()]
-  extensions = glob.glob('extensions/*/0.1/validator-*.protoascii')
+  extensions = glob.glob('extensions/*/(0.1|1.0)/validator-*.protoascii')
```

Comment:
Even though amp-sticky-ad 0.1 is deprecated it is still valid and therefore must still be tested.